<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FactuurRegel</class>
 <widget class="QDialog" name="FactuurRegel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frameArtikel">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>5</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="tbArtikelnummer">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>5</y>
      <width>181</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMask">
     <string>N.N.NN.NN</string>
    </property>
    <property name="text">
     <string>0.0.00.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxProduct">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>45</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxSoort">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>75</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbSoort">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>61</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Soort:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxDetail">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>105</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxSpecificatie">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>135</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbDetail">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>61</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Detail:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSpecificatie">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Specificatie:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbProduct">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>55</y>
      <width>61</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Product:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZoek_Art">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>45</y>
      <width>61</width>
      <height>56</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/zoom.png</normaloff>:/icons/icons/zoom.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWis_Art">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>105</y>
      <width>61</width>
      <height>56</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/draw_eraser.png</normaloff>:/icons/icons/draw_eraser.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="tbArtikelNaam">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>45</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>tbArtikelNaam</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbArtikelnummer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Benaming:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbArtikelCode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Artikelcode:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbArtikelCode">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>15</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>tbArtikelCode</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbEenheidsprijs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Eenheidsprijs:&lt;br/&gt;(excl BTW)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbEenheidsprijs">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>75</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>tbEenheidsprijs</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBTW">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BTW:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxBTW">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>75</y>
     <width>86</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbHoeveelheid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hoeveelheid:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbHoeveelheid">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>105</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>tbHoeveelheid</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbEenheid">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Eenheid:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbPromo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Promo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxEenheid">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>105</y>
     <width>136</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbKorting">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Korting in%:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbKorting">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>135</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>tbKorting</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_code">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 241, 241);</string>
   </property>
   <property name="text">
    <string>tb_code</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_omschr">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>206</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 241, 241);</string>
   </property>
   <property name="text">
    <string>tb_omschr</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_EP">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>210</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 241, 241);</string>
   </property>
   <property name="text">
    <string>tb_EP</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_Hoeveel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 241, 241);</string>
   </property>
   <property name="text">
    <string>tb_Hoeveel</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_Eenh">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>210</y>
     <width>46</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 241, 241);</string>
   </property>
   <property name="text">
    <string>tb_Eenh</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_Pr">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 241, 241);</string>
   </property>
   <property name="text">
    <string>tb_Pr</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_Ko">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>210</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 241, 241);</string>
   </property>
   <property name="text">
    <string>tb_Ko</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_PrKo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>210</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 241, 241);</string>
   </property>
   <property name="text">
    <string>tb_PrKo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_BTW_P">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>210</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 241, 241);</string>
   </property>
   <property name="text">
    <string>tb_BTW_P</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_BTW_B">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>210</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 241, 241);</string>
   </property>
   <property name="text">
    <string>tb_BTW_B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_LP">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 241, 241);</string>
   </property>
   <property name="text">
    <string>tb_LP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_code">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>195</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_omschr">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>195</y>
     <width>106</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Omschrijving:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_EP">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>195</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Eenh.Pr.:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Hoeveel">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>195</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Aant.:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Eenh">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>195</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Eenh.:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pr">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>195</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bruto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Ko">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>195</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Korting:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_PrKo">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>195</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pr.Excl.:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_BTWP">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>195</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BTW%:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_BTWB">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>195</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BTW B.:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_LP">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>195</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pr.Incl.:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOvernemen">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>245</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Overnemen</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnnuleren">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>245</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuleren</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWissen">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>245</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Regel Wissen</string>
   </property>
   <property name="text">
    <string>Regel wissen</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tbArtikelnummer</tabstop>
  <tabstop>tbArtikelCode</tabstop>
  <tabstop>tbArtikelNaam</tabstop>
  <tabstop>tbEenheidsprijs</tabstop>
  <tabstop>tbHoeveelheid</tabstop>
  <tabstop>tbKorting</tabstop>
  <tabstop>cbxBTW</tabstop>
  <tabstop>cbxEenheid</tabstop>
  <tabstop>btnWis_Art</tabstop>
  <tabstop>cbPromo</tabstop>
  <tabstop>tb_code</tabstop>
  <tabstop>tb_omschr</tabstop>
  <tabstop>tb_EP</tabstop>
  <tabstop>tb_Hoeveel</tabstop>
  <tabstop>tb_Eenh</tabstop>
  <tabstop>tb_Pr</tabstop>
  <tabstop>tb_Ko</tabstop>
  <tabstop>tb_PrKo</tabstop>
  <tabstop>tb_BTW_P</tabstop>
  <tabstop>tb_BTW_B</tabstop>
  <tabstop>tb_LP</tabstop>
  <tabstop>btnAnnuleren</tabstop>
  <tabstop>btnOvernemen</tabstop>
  <tabstop>cbxProduct</tabstop>
  <tabstop>cbxSoort</tabstop>
  <tabstop>cbxDetail</tabstop>
  <tabstop>cbxSpecificatie</tabstop>
  <tabstop>btnZoek_Art</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
